components/tabs/tab.js

```diff
@@ -21,20 +21,22 @@ class Tab extends TabMixin(LitElement) {
 	static get styles() {
 		const styles = [ css`
 			.d2l-tab-text {
-				margin: 0.5rem;
```

Comment:
I moved the margins out into the `TabMixin` since that felt like more of a general shared thing, and also since the selected-indicator margins are there as well

Reply:
Agreed. I think the subclass should only handle layout of its internals.

Reply:
Another option would be for `d2l-tabs` to control it via `::slotted(role="tab")`.